README.md

```diff
@@ -355,7 +355,9 @@ nano.db.list().then((body) => {
 Lists all the CouchDB databases as a stream:
 
 ```js
-nano.db.list().pipe(process.stdout);
+nano.db.list()
```

Comment:
I think this should be `nano.db.listAsStream()`. An error in the original, not in your PR.

Reply:
I missed these too. Corrected in the latest commit.